hooks/build

```diff
@@ -401,34 +372,30 @@ git ls-files -z -- ./ ':!:airflow/www/static/docs' | filterout_deleted_files | x
 # and this deals with directories
```

Comment:
Can you remove the exclusion I added on the previous `git ls-files` lines now you've removed the symlink please?

Reply:
done